.gitignore

```diff
@@ -0,0 +1,2 @@
+src/data/netflix_movies.txt
```

Comment:
Remember to update this file when we change the input/output directory locations.

I recommend

```
data/
out/
```

and just ignore those two directories

Reply:
Updated the .gitignore file to remove  "MediaBridge" from the directory path so that it's no longer treating it as a folder that stores "data" and out" but as the root of the repo.